### ISSUE TITLE
Setting Filters for messages on "Raganork-md" Bot

### ISSUE THREAD

**Issue by SinethFernando** [souravkl11/raganork-md#39]
I would like to know the **FORMAT** on how to set a message to be automatically sent to a group or a contact by the bot when the user clicks on a button I created.

ex:- I have already set a welcome message to be sent to a group when a user is added to the group newly. As mentioned in your Wiki, I have created a custom button, and would like the bot to send a specific message (pre-written) to the group when the user clicks on it.

- _Such as "Rules" button to show the Rules of the Group._

In summary, all I want is the **FORMAT** of creating a Filter in a Group.

Thank you!

**Comment by souravkl11** [comment]
> I would like to know the **FORMAT** on how to set a message to be automatically sent to a group or a contact by the bot when the user clicks on a button I created.
> 
> ex:- I have already set a welcome message to be sent to a group when a user is added to the group newly. As mentioned in your Wiki, I have created a custom button, and would like the bot to send a specific message (pre-written) to the group when the user clicks on it.
> 
> * _Such as "Rules" button to show the Rules of the Group._
> 
> In summary, all I want is the **FORMAT** of creating a Filter in a Group.
> 
> Thank you!

Hey @SinethFernando, sorry for the late reply ; )
You can set this using filters! Just use:

.filter "Rules" "YOUR RULES HERE".
                   Or 
You can reply to a message and type: .filter "Rules"

**Comment by SinethFernando** [comment]
Thank you so much @souravkl11 ! It works like a charm! Can I also know how to **delete a Filter** Just in case if required?
 Thank you!
